plugins/catalog-import/src/util/useGithubRepos.ts

```diff
@@ -15,22 +15,39 @@
  */
 
 import * as YAML from 'yaml';
-import { useApi } from '@backstage/core';
+import { useApi, configApiRef } from '@backstage/core';
 import { catalogImportApiRef } from '../api/CatalogImportApi';
 import { ConfigSpec } from '../components/ImportComponentPage';
 
+// TODO: (O5ten) Refactor into a core API instead of direct usage like this
+// https://github.com/backstage/backstage/pull/3613#issuecomment-7408929430
+import { readGitHubIntegrationConfigs } from '@backstage/integration';
+
 export function useGithubRepos() {
   const api = useApi(catalogImportApiRef);
+  const config = useApi(configApiRef);
 
   const submitPrToRepo = async (selectedRepo: ConfigSpec) => {
-    const [ownerName, repoName] = selectedRepo.location.split('/').slice(-2);
+    const [hostname, ownerName, repoName] = selectedRepo.location
+      .split('/')
+      .slice(-3);
```

Comment:
@O5ten you still have this here instead of git-url-parse - maybe you forgot to push the change?